Lombiq.Tests.UI/Pages/OrchardCoreLoginPage.cs

```diff
@@ -66,7 +67,16 @@ public async Task<_> LogInWithAsync(UITestContext context, string userName, stri
             await context.ClickAndFillInWithScriptAsync(passwordBy, password);
         }
 
-        await context.ClickReliablyOnSubmitAsync();
+        var buttonBy = ByHelper.ButtonText("Log in");
```

Comment:
I just realized this, but this could be "Sing in" in some other places, so it was better when it was generalized. This caused a failing test in another project.

Reply:
We need to use `ClickReliablyOnUntilNavigationHasOccurredAsync` here so it is reliable, but feel free to change `buttonBy` Best would be to use the `By` of `ClickReliablyOnSubmitAsync`, like from a static readonly field.

Reply:
> I just realized this, but this could be "**Sing** in" in some other places

omg new authentication method just dropped!

![image](https://github.com/user-attachments/assets/c6b96256-853f-42a7-8f8f-0b0d2f3902a9)
mfw I can't hit the tone to get into my account

Reply:
Another is login by piano: https://imgur.com/favorite-of-these-comics-SYvyZ